### PR TITLE
ARMOCP-468: Adding Arm Azure IPI jobs

### PR DESCRIPTION
**Pull request by deepsm007**
Enabling Arm Azure IPI CI jobs for ovn 

/cc @aleskandro @Prashanth684

### CODE REVIEW

**Comment by deepsm007** [comment]
Depends on https://github.com/openshift/release/pull/33149

File: ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.12.yaml [hunk 1]
```diff
@@ -265,6 +265,16 @@ tests:
...
       COMPUTE_NODE_TYPE: m6g.xlarge
     workflow: openshift-e2e-aws-techpreview-serial
+- as: ocp-e2e-azure-ovn-arm64
+  interval: 72h
+  steps:
+    cluster_profile: azure-arm64
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
+    env:
+      BASE_DOMAIN: arm-ci.azure.devcluster.openshift.com
```
Comment (lwan-wanglin) [inline ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.12.yaml:275]: need to add `OCP_ARCH: arm64` in env, default is amd64.

Comment (deepsm007) [reply ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.12.yaml:275]: Yes, I was waiting for the other PR to merge.

**Review by wking — APPROVED** [review]
/lgtm

**Pull request merged**